### PR TITLE
[ASM] Expandr 2086

### PR DESCRIPTION
**Pull request by johnnywilkes**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-hq.paloaltonetworks.local/browse/EXPANDR-2086
https://jira-hq.paloaltonetworks.local/browse/CRTX-72274
https://jira-hq.paloaltonetworks.local/browse/EXPANDR-2066

## Description
See description of Bug https://jira-hq.paloaltonetworks.local/browse/CRTX-72274
This is a stop-gap solution until that is fixed
added "# flake8: noqa" to top of file.  Open to other options

see other changes in https://jira-hq.paloaltonetworks.local/browse/EXPANDR-2066

## Screenshots
N/A

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [X] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [X] No

## Must have
- [ ] Tests
- [X] Documentation

### CODE REVIEW

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_4_1.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+
+#### Scripts
+##### GenerateASMReport
+- Fixed a parsing issue that was causing script to fail.
+- Updated docker image.
```
Comment (adi88d) [inline Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_4_1.md:5]: remove the GenerateASMReport from here

**Review by adi88d — CHANGES_REQUESTED** [review]
Hi @johnnywilkes and thank you for your contribution! it looks great!
Please revert the changes you made in the GenerateASMReport script, we aware of the issue and we already fixed it

File: Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_4_1.md (hunk 1 above)
Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_4_1.md:5]: @adi88d, I am still seeing the issue from https://jira-hq.paloaltonetworks.local/browse/XSUP-20075 and would prefer to keep the changes in there.  I am not sure when that jira ticket is expected to be complete so we would like this as a stop gap solution.

Comment (johnnywilkes) [reply Packs/CortexAttackSurfaceManagement/ReleaseNotes/1_4_1.md]: @adi88d, i undid the changes to the automation but figured i would update docker image.  Let me know what you think.

**Comment by ShirleyDenkberg** [comment]
@adi88d Doc review completed.

**Comment by johnnywilkes** [comment]
@adi88d , anything waiting to merge this?

**Review by adi88d — APPROVED** [review]

**Comment by johnnywilkes** [comment]
@adi88d, where is the link to the 2nd PR for this (internal PR)?

**Comment by ShacharKidor** [comment]
> @adi88d, where is the link to the 2nd PR for this (internal PR)?

@johnnywilkes - [link to internal PR](https://github.com/demisto/content/pull/24153).

**Pull request merged**